### PR TITLE
CROWBAR_EXPERIMENTAL is not longer needed for Cloud6

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

**Comment by rsalevsky** [comment]
I would just keep it. We could have experimental features again and then we would have to add it again.

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -92,8 +92,6 @@ export want_postgresql=${want_postgresql:-1}
 [ -z "$want_test_updates" -a -n "$TESTHEAD" ] && export want_test_updates=1
 [ "$libvirt_type" = hyperv ] && export wanthyperv=1
 [ "$libvirt_type" = xen ] && export wantxenpv=1 # xenhvm is broken anyway
-# FIXME: delete this after cloud 7 is released
-iscloudver 6plus && export CROWBAR_EXPERIMENTAL=true
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: maybe change this to ```iscloudver 7plus``` and change the comment to cloud 8 respectively? would make more sense to me.

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: Well, what meaning should it have?
`iscloudver 7plus` is a normal setup supported by crowbarctl. And check for `CROWBAR_EXPERIMENTA` was removed from crowbar-client code anyway, so I do not really see a point

Comment (rsalevsky) [inline scripts/qa_crowbarsetup.sh]: `CROWBAR_EXPERIMENTAL` was not removed only the part where we used it

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: And because we do not use it in any way there's no point for exporting the value here

**Pull request closed without merging**